sdk/python/feast/infra/registry/snowflake.py

```diff
@@ -418,7 +418,7 @@ def _delete_object(
             """
             cursor = execute_snowflake_statement(conn, query)
 
-            if cursor.rowcount < 1 and not_found_exception:
+            if cursor.rowcount < 1 and not_found_exception:  # type: ignore
```

Comment:
I tried this and can pass the mypy check:
`  if cursor.rowcount and (cursor.rowcount < 1) and not_found_exception:`

Reply:
You're right, this is due to another change I did to the mypy config: `--follow-imports=skip`. 

run the below command to see the error:
python -m mypy --exclude=/tests/ sdk/python/feast/infra/registry/snowflake.py you will see an error.
```
`sdk/python/feast/infra/registry/snowflake.py:421: error: Unsupported operand types for > ("int" and "None")  [operator]
```

Reply:
essentially cursor.rowcount could return none from snowflake's signature
```
    @property
    def rowcount(self) -> int | None:
        return self._total_rowcount if self._total_rowcount >= 0 else None
 ```